SimCalorimetry/HcalTrigPrimProducers/python/hcaltpdigi_cfi.py

```diff
@@ -17,34 +17,34 @@
     weightsQIE11 = cms.untracked.PSet(**dict([
-        ("1",  cms.untracked.vdouble(1.0, 1.0)),
-        ("2",  cms.untracked.vdouble(1.0, 1.0)),
-        ("3",  cms.untracked.vdouble(1.0, 1.0)),
-        ("4",  cms.untracked.vdouble(1.0, 1.0)),
-        ("5",  cms.untracked.vdouble(1.0, 1.0)),
-        ("6",  cms.untracked.vdouble(1.0, 1.0)),
-        ("7",  cms.untracked.vdouble(1.0, 1.0)),
-        ("8",  cms.untracked.vdouble(1.0, 1.0)),
-        ("9",  cms.untracked.vdouble(1.0, 1.0)),
-        ("10", cms.untracked.vdouble(1.0, 1.0)),
-        ("11", cms.untracked.vdouble(1.0, 1.0)),
-        ("12", cms.untracked.vdouble(1.0, 1.0)),
-        ("13", cms.untracked.vdouble(1.0, 1.0)),
-        ("14", cms.untracked.vdouble(1.0, 1.0)),
-        ("15", cms.untracked.vdouble(1.0, 1.0)),
-        ("16", cms.untracked.vdouble(1.0, 1.0)),
-        ("17", cms.untracked.vdouble(1.0, 1.0)),
-        ("18", cms.untracked.vdouble(1.0, 1.0)),
-        ("19", cms.untracked.vdouble(1.0, 1.0)),
-        ("20", cms.untracked.vdouble(1.0, 1.0)),
-        ("21", cms.untracked.vdouble(1.0, 1.0)),
-        ("22", cms.untracked.vdouble(1.0, 1.0)),
-        ("23", cms.untracked.vdouble(1.0, 1.0)),
-        ("24", cms.untracked.vdouble(1.0, 1.0)),
-        ("25", cms.untracked.vdouble(1.0, 1.0)),
-        ("26", cms.untracked.vdouble(1.0, 1.0)),
-        ("27", cms.untracked.vdouble(1.0, 1.0)),
-        ("28", cms.untracked.vdouble(1.0, 1.0))
+        ("ieta1",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta2",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta3",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta4",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta5",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta6",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta7",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta8",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta9",  cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta10", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta11", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta12", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta13", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta14", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta15", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta16", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta17", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta18", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta19", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta20", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta21", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta22", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta23", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta24", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta25", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta26", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta27", cms.untracked.vdouble(1.0, 1.0)),
+        ("ieta28", cms.untracked.vdouble(1.0, 1.0))
         ])
```

Comment:
```suggestion
    weightsQIE11 = cms.untracked.PSet(
        ieta1 =  cms.untracked.vdouble(1.0, 1.0),
        ieta2 =  cms.untracked.vdouble(1.0, 1.0),
        ieta3 =  cms.untracked.vdouble(1.0, 1.0),
        ieta4 =  cms.untracked.vdouble(1.0, 1.0),
        ieta5 =  cms.untracked.vdouble(1.0, 1.0),
        ieta6 =  cms.untracked.vdouble(1.0, 1.0),
        ieta7 =  cms.untracked.vdouble(1.0, 1.0),
        ieta8 =  cms.untracked.vdouble(1.0, 1.0),
        ieta9 =  cms.untracked.vdouble(1.0, 1.0),
        ieta10 = cms.untracked.vdouble(1.0, 1.0),
        ieta11 = cms.untracked.vdouble(1.0, 1.0),
        ieta12 = cms.untracked.vdouble(1.0, 1.0),
        ieta13 = cms.untracked.vdouble(1.0, 1.0),
        ieta14 = cms.untracked.vdouble(1.0, 1.0),
        ieta15 = cms.untracked.vdouble(1.0, 1.0),
        ieta16 = cms.untracked.vdouble(1.0, 1.0),
        ieta17 = cms.untracked.vdouble(1.0, 1.0),
        ieta18 = cms.untracked.vdouble(1.0, 1.0),
        ieta19 = cms.untracked.vdouble(1.0, 1.0),
        ieta20 = cms.untracked.vdouble(1.0, 1.0),
        ieta21 = cms.untracked.vdouble(1.0, 1.0),
        ieta22 = cms.untracked.vdouble(1.0, 1.0),
        ieta23 = cms.untracked.vdouble(1.0, 1.0),
        ieta24 = cms.untracked.vdouble(1.0, 1.0),
        ieta25 = cms.untracked.vdouble(1.0, 1.0),
        ieta26 = cms.untracked.vdouble(1.0, 1.0),
        ieta27 = cms.untracked.vdouble(1.0, 1.0),
        ieta28 = cms.untracked.vdouble(1.0, 1.0)
    )
```
What about this?

Reply:
Hmm, so the key must be a string, otherwise it gets interpreted as a variable.

Reply:
It should work exactly in the same way in both cases. 
I checked it with
```
import FWCore.ParameterSet.Config as cms

weightsQIE11 = cms.untracked.PSet(**dict([
        ("ieta1",  cms.untracked.vdouble(1.0, 1.0)),
        ("ieta2",  cms.untracked.vdouble(1.0, 1.0)),
        ("ieta3",  cms.untracked.vdouble(1.0, 1.0)),
        ]))

weightsQIE11_new = cms.untracked.PSet(
        ieta1 = cms.untracked.vdouble(1.0, 1.0),
        ieta2 = cms.untracked.vdouble(1.0, 1.0),
        ieta3 = cms.untracked.vdouble(1.0, 1.0),
        )


print(weightsQIE11.dumpPython())
print(weightsQIE11_new.dumpPython())
print()
print(weightsQIE11.ieta1)
print(weightsQIE11_new.ieta1)
print()
print(getattr(weightsQIE11,"ieta1"))
print(getattr(weightsQIE11_new,"ieta1"))
```

Reply:
Ah, I see. I will make this simplification.

Reply:
Addressed in 9877930.